server/api/appointments/notes/notes.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
I'd also like feedback on how I set this file up / where I put it. We haven't really been putting scripts like this in the api folder (and by "scripts like this", I mean a script that accepts an action and then delegates what to do based off of that parameter). So I'm not sure if this breaks our current architectural style, or if there would be a better place to put it.

Reply:
@CollinsSpencer

Reply:
Upon further thought, I think this is an ok place for it